src/index.html

```diff
@@ -1,14 +1,66 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <title>Moyo header</title>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header__logo" >
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="image"
+          >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover"
+              >
+              laptops & computers
+            </a>
```

Comment:
```suggestion
            <a
              href="#"
              class="nav__link"
              data-qa="hover"
            >
              laptops & computers
            </a>
```
check your code style